wurst/_handles/TimerDialog.wurst

```diff
@@ -0,0 +1,30 @@
+package TimerDialog
+import NoWurst
+
+public function timer.createTimerDialog() returns timerdialog
+    return CreateTimerDialog(this)
+
+public function timerdialog.destr() 
+    DestroyTimerDialog(this)
+
+public function timerdialog.setTitle(string title) 
+    TimerDialogSetTitle(this, title)
+
+public function timerdialog.setTitleColor(int red, int green, int blue, int alpha) 
```

Comment:
please add an overloaded version of this function taking a colorA tuple.

Reply:
Done